src/client/api/v1/get_node_info.c

```diff
@@ -147,6 +147,13 @@ int deser_node_info(char const *const j_str, res_node_info_t *res) {
       ret = -1;
       goto end;
     }
+
+    // TODO: gets features
+    // if ((ret = json_string_array_to_utarray(data_obj, key_features, res->u.output_node_info->features)) != 0) {
```

Comment:
@oopsmonk how do you suggest to implement this?
your original implementation uses `char features[128]`, however how do we handle multiple features?
is `utarray` a good approach, or am I missing something from the `char` array approach?

Reply:
the `res_node_info_t` is outdated. utarray can be used instead of char array.